dlc-wallet/src/main/resources/sqlite/dlc/migration/V5__serialization_version_global_dlc_data.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "global_dlc_data"
+    ADD COLUMN "serialization_version" VARCHAR(254);
```

Comment:
I think it'd make sense to have this be `NOT NULL` with a default of Alpha and then update the correct ones accordingly. That way the schema enforces that a `serialization_version` must be set